tezos-sig-gating-contracts/scripts/tests/gatednftclaimer.spec.ts

```diff
@@ -79,13 +101,83 @@ describe(`GatedNftMinterSimple`, function () {
     expect(ownerAsset3).to.be.undefined;
   });
 
+  function decodeMintEvent(data: InternalOperationResult) {
```

Comment:
so taquito hasn't released anything like that yet? Otherwise this looks good to me